pkg/rpc/client/nep5.go

```diff
@@ -18,9 +18,9 @@ import (
 
 var (
 	// NeoContractHash is a hash of the NEO native contract.
-	NeoContractHash, _ = util.Uint160DecodeStringBE("3b7d3711c6f0ccf9b1dca903d1bfa1d896f1238c")
+	NeoContractHash, _ = util.Uint160DecodeStringBE("897720d8cd76f4f00abfa37c0edd889c208fde9b")
```

Comment:
I'd better use `Uint160DecodeStringLE` then. Let me check it again, though.

Reply:
```
    helper_test.go:177: native GAS hash: 3b7d3711c6f0ccf9b1dca903d1bfa1d896f1238c
    helper_test.go:178: native NEO hash: 897720d8cd76f4f00abfa37c0edd889c208fde9b
```
Yep, these are BE variants. But still I think it's more appropriate to store LE as strings, these are human-facing variants and BE values are not copy-pasteable for some other purposes.